docs/model/introduction-to-model-architecture.md

```diff
@@ -193,6 +195,14 @@ class CartFacade
 ```
 
 ## Cooperation of layers
-The controller handles the request (eg. [saved data](entities.md#entity-data) from form) and passes data to the facade.
+The controller handles the request (eg. [saved data](../introduction/entities.md#entity-data) from form) and passes data to the facade.
 The facade receives data from the controller and requests appropriate entities from the repository.
 Entities and supporting classes (like recalculators, schedulers) processes data and returns output to the facade, that persist it by entity manager.
+
+## Model extension
+Entity extension is described in [Entity Extension article](../extensibility/entity-extension.md).
+
+Other parts of a model are extended by standard inheritance.
```

Comment:
I'm not sure whether class inheritance can be described as standard for extension. Maybe we could add a bit of an explanation:

Other parts of a model can be extended by class inheritance and adding an alias to your `services.yml`, eg.:
```yaml
services:
    Shopsys\FrameworkBundle\Model\Article\ArticleFacade: '@Shopsys\ShopBundle\Model\Article\ArticleFacade'
```